bp_common/test/src/perch/riscv.ld

```diff
@@ -0,0 +1,146 @@
+OUTPUT_ARCH( "riscv" )
```

Comment:
Just to make sure, this handles both C and C++ programs? And the extra C++ stuff just gets trimmed when we link it to our C bare-metal stuff?

Reply:
Yes. From my understanding, these init calls call a series of constructor initializers in the C++ case. If there are no constructors then it basically doesn't do anything.